src/services/files.service.js

```diff
@@ -0,0 +1,66 @@
+import multer from 'multer';
+import fs from 'fs';
+import aws from 'aws-sdk';
+import config from '../config';
+
+/**
+ * Files service
+ */
+class fileService {
+  /**
+  * @param {String} field
+  * @returns {object} file
+  */
+  upload(field) {
```

Comment:
Does this handle multiple images?

Reply:
This one upload a single image,  we can implement another function to upload multiple images